main.go

```diff
@@ -27,7 +24,6 @@ func main() {
 	// so while this isn't pretty, we'll use a separate package to carry state from here. If you
 	// know of a better way of doing this, don't hesitate to fix this.
 	embeds.EmbedBundleFS = bundle
-	embeds.ASTSchema = schema
```

Comment:
@charlieegan3 as you requested in a previous PR :) Moved to the embeds directory.

Reply:
👏